ide/image/src/org/netbeans/modules/image/ImageOpenSupport.java

```diff
@@ -89,20 +91,16 @@ public void fileChanged(final FileEvent evt) {
                         entry.getFile().addFileChangeListener(this);
                         return;
                     }
-                    
+
                     if (evt.getTime() > lastSaveTime) {
                         lastSaveTime = System.currentTimeMillis();
-                        
+
                         // Post in new task.
                         if(reloadTask == null || reloadTask.isFinished()) {
-                        
-                            reloadTask = RequestProcessor.getDefault().post(
-                                new Runnable() {
-                                    public void run() {
-                                        reload(evt);
-                                    }
-                                }
-                            );
+
+                            reloadTask = RequestProcessor.getDefault().post(() -> {
```

Comment:
how about  ?
reloadTask = RequestProcessor.getDefault().post(this::reload);

Reply:
Thank you, valid idea. In the end the two should be more or less equivalent and runtime is dominated by the actual work, so I'll not respin this patch.